edit-post/api/index.js

```diff
@@ -1,4 +1,11 @@
+import { registerPlugin } from './plugin';
+import { PluginSidebarFill as PluginSidebar } from './components/plugin-sidebar';
+
+const __experimental = {
+	registerPlugin,
+	PluginSidebar,
+};
+
 export {
-	registerSidebar as __experimentalRegisterSidebar,
```

Comment:
While it is clearly flagged as `__experimental`, is there any way we can [`deprecated`](https://github.com/WordPress/gutenberg/blob/master/utils/deprecation.js) this easily with  a transparent upgrade path?